studio/components/interfaces/Organization/GeneralSettings/MigrateOrganizationBillingButton.tsx

```diff
@@ -108,9 +113,38 @@ const MigrateOrganizationBillingButton = observer(() => {
   return (
     <>
       <div>
-        <Button loading={!organization?.slug} onClick={() => setIsOpen(true)} type="primary">
-          Migrate organization
-        </Button>
+        <Tooltip.Root delayDuration={0}>
+          <Tooltip.Trigger>
+            <Button
+              loading={!organization?.slug}
+              onClick={toggle}
+              type="primary"
+              disabled={!canMigrateOrg || disableOrgBillingMigration}
+            >
+              Migrate organization
+            </Button>
+          </Tooltip.Trigger>
+          {!canMigrateOrg ||
+            (disableOrgBillingMigration && (
```

Comment:
I think these parens are wrong? Should it be: `(!canMigrateOrg || disableOrgBillingMigration) &&`?

Reply:
Good catch, fixing